pandas/tslib.pyx

```diff
@@ -4822,8 +4833,23 @@ def dates_normalized(ndarray[int64_t] stamps, tz=None):
 #----------------------------------------------------------------------
 
 def isleapyear(int64_t year):
```

Comment:
I don't think any reason to deprecate this; by-definition its internal if in tslib.pyx